presets/all-packages/README.md

```diff
@@ -0,0 +1,81 @@
+# @commercetools-test-data/cart
```

Comment:
Is this a typo?

Reply:
Yes.

Updated here: 9b894da0e0dde37d04c180a62347d228b9032a0b